source/windows/bcrypt_hash.c

```diff
@@ -92,6 +114,35 @@ struct aws_hash *aws_sha256_default_new(struct aws_allocator *allocator) {
     return &bcrypt_hash->hash;
 }
 
+struct aws_hash *aws_sha1_default_new(struct aws_allocator *allocator) {
+    aws_thread_call_once(&s_sha1_once, s_load_sha1_alg_handle, NULL);
+
+    struct bcrypt_hash_handle *bcrypt_hash;
```

Comment:
Initialize both of these to NULL. If aws_mem_acquire_many were to fail, these could be their initial random values. (https://github.com/awslabs/aws-c-common/blob/main/source/allocator.c#L190).